recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+channel_sources:
+  - pytorch,conda-forge,defaults
```

Comment:
We don't allow channels outside of `conda-forge, defaults` to be in `channel_sources` in `conda-forge`.

Reply:
OK, I will try w/o pytorch channel

Reply:
@isuruf   but what is the purpose? of
~~~
Specify the feedstock channel and label: Optionally, you can specify source channels and choose a channel to upload to in recipe/conda_build_config.yaml.

channel_sources:
  - mysourcechannel1,mysourcechannel2,conda-forge,defaults
channel_targets:
  - target_channel target_label
~~~ 
from 

https://github.com/conda-forge/conda-smithy#making-a-new-feedstock

Reply:
There are other github orgs (other than conda-forge) using conda-smithy to generate feedstocks. Only conda-forge feedstocks disallow channels outside conda-forge.